js/Plan.jsx

```diff
@@ -22,7 +20,8 @@ var store = Redux.createStore(
 
 var Plan = function(props) {
 
-	window.api_url = props.model.APIEndpoint;
+	window.api_url = props.model.api_endpoint;
```

Comment:
oh sorry, the comment was to the line below: if you store the api_auth in window, what will happen if you have multiple dispatchers? Seems to me like the token state will go out of sync, especially that we are using the dispatcher ClassName in the token name?